polynote-runtime/src/main/scala/polynote/runtime/python/pandas/PandasHandle.scala

```diff
@@ -22,7 +22,7 @@ class PandasHandle(val handle: Int, df: PythonObject) extends StreamingDataRepr.
     case "boolean" => Some(BoolType)
     case "float64" => Some(DoubleType)
     case "float32" => Some(FloatType)
-    case "string" | "category" => Some(StringType)
+    case "string" | "category" | "object" => Some(StringType)
```

Comment:
The problem here is that if it's `StringType`, then once we get to `fieldEncoder` below, it's going to try casting directly to `String` and that won't work. We'll need some additional logic here to thread through some optional pre-conversion on the jep side (e.g. so we can call `str` on the object if it's not actually a `string` or `category`).

Reply:
> The problem here is that if it's `StringType`, then once we get to `fieldEncoder` below, it's going to try casting directly to `String` and that won't work. We'll need some additional logic here to thread through some optional pre-conversion on the jep side (e.g. so we can call `str` on the object if it's not actually a `string` or `category`).

Thanks for reply. Yes, I see, this issue is related with many things, I think I can first close this Pull request, and get more familiar with polynote's source code, and then, make a more thoughtful pull request for this "object" column type support.